### PR TITLE
unblock EXP-96

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

Temporary disable auto-attach of the async-profiler to prevent JVM crash. See [internal Slack thread](https://gitpod.slack.com/archives/C02BRJLGPGF/p1690795735689349?thread_ts=1687385739.886699&cid=C02BRJLGPGF)

<details>
<summary>Summary generated by Copilot</summary>

### <samp>🤖 Generated by Copilot at 85573e4</samp>

Disable async-profiler attachment in JetBrains IDEs to avoid JVM crashes. Modify `updateVMOptions` in `components/ide/jetbrains/launcher/main.go` to add a new VM option based on an environment variable.

</details>

## Related Issue(s)

Related to EXP-96

## How to test

Smoke a workspace startup with IntelliJ (stable and latest).

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - ak-jb-disac8223fdb64</li>
	<li><b>🔗 URL</b> - <a href="https://ak-jb-disac8223fdb64.preview.gitpod-dev.com/workspaces" target="_blank">ak-jb-disac8223fdb64.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - ak-jb_disable_freeze_profiling-gha.14413</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

**Review by geropl — APPROVED** [review]
Don't know how to test, approving to unblock. :white_check_mark: 

/hold

**Review by filiptronicek — APPROVED** [review]
Connecting works on both stable and latest. 

image

Wow, the new UI JB has is really nice.

**Comment by akosyakov** [comment]
@filiptronicek thanks for testing, was about doing it 😄 

/unhold

**Pull request merged**
